### ISSUE TITLE
Remove extra columns from attempt report once the model is validated

### ISSUE THREAD

**Issue by adamfranco** [middlebury/moodle-mod_adaptivequiz#1]
The following attempt-report columns can be removed once the operation of the system is fully validated:
- Question Difficulty (logits)
- Difficulty Sum
- Measured Ability (logits)
- Standard Error (± logits)

For now they can be left in place to aid operational analysis.
